core-build/gulp-core-build-webpack/src/WebpackTask.ts

```diff
@@ -175,9 +175,9 @@ export class WebpackTask<TExtendedConfig = {}> extends GulpTask<IWebpackTaskConf
                 child.chunks.forEach(chunk => {
                   if (chunk.files && this.taskConfig.printStats) {
                     chunk.files.forEach(file => (
-                      this.log(`Bundled: '${gutil.colors.cyan(path.basename(file))}', ` +
-                                `size: ${gutil.colors.magenta(chunk.size)} bytes, ` +
-                                `took ${gutil.colors.magenta(duration)} ms.`)
+                      this.log(`Bundled: '${colors.cyan(path.basename(file))}', ` +
+                                `size: ${colors.magenta(chunk.size)} bytes, ` +
+                                `took ${colors.magenta(duration.toString(10))} ms.`)
```

Comment:
>duration.toString(10) [](start = 55, length = 21)

Unrelated?

Reply:
It is related. We can't pass a number into colors.